docker/README.md

```diff
@@ -121,7 +121,7 @@ This project is licensed under the MIT License - see the [LICENSE.md](https://gi
 [vcf-repo]: https://github.com/TileDB-Inc/TileDB-VCF
 [docker]: https://hub.docker.com/u/tiledb
 [tiledb]: https://tiledb.com
```

Comment:
maybe link to https://github.com/TileDB-Inc/TileDB?

Reply:
If we do, we should change the text from `TileDB` to `TileDB Embedded` at the top.